src/config/dynamic-theme-fixes.config

```diff
@@ -14346,6 +14346,26 @@ CSS
 
 ================================
 
+purerave.com
+
+CSS
+:root {
+    --border: 1px solid rgba(0, 90, 155, 1) !important;
+    --box-shadow: 0 1px 8px 0 rgba(0, 90, 155, 1) !important;
```

Comment:
Those variables should be overwritten by dark reader already(to use the Dark reader modified version) on the elements that uses this variables. Are these modifications necessary?

Reply:
You're correct, thank you! I was confused that I needed it too, I must have made a mistake when testing because it works without now.

Reply:
Fixed in https://github.com/darkreader/darkreader/pull/9198/commits/a52b1496f736fcb5e17236016e7ea4771d03e629